index.js

```diff
@@ -1,5 +1,276 @@
+
+const workHeaderSubSection = () => {
+  const workHeaderSectionTitle = document.createElement('div');
+  workHeaderSectionTitle.className = 'works-title-block';
+  workHeaderSectionTitle.innerHTML = `
+    <h3 class="title flex">My Recent Works</h3> 
+      <div class="divider_1"> 
+        <img class="divider_2" src="images/desk_tab.png" alt="vector">
+      </div>
+    </div>
+    <div class="crossline">
+      <img src="images/cross_vector.png" alt="vector">
+    </div>`;
+  workHeaderSection.appendChild(workHeaderSectionTitle);
+
+  const yogaImageCard = document.createElement('div');
+  yogaImageCard.className = 'works-card';
+  yogaImageCard.innerHTML = `
+    <div class="yoga-placeholder">
+      <img src="images/yoga-image.png" alt="yoga">
+    </div>
+    <h3 id="bold-text" class="flex">Multi-Post Stories</h3>
+    <p id="works-description">
+      A daily selection of privately personalized reads; no accounts or sign-ups required.
+      has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard
+      dummy text.
+    </p>
+    <ul id="skillset" class="flex">
+      <li>CSS</li>
+      <li>HTML</li>
+      <li>JAVASCRIPT</li>
+    </ul>
+    <button type="button" class="btn-type-a flex">See project</button>`;
+  workHeaderSection.appendChild(yogaImageCard);
+};
+workHeaderSubSection();
+
+function worksDesktop() {
+  const worksCardDesktop = document.createElement('div');
+  worksCardDesktop.className = 'works-card-desktop';
+  worksCardDesktop.innerHTML = `
+  <img src="images/yoga-image.png" class="project-cards flex">
+  <div class="extras side-text">
+    <h3 id="bold-text">Multi-Post Stories</h3>
+    <p id="works-description">
+          A daily selection of privately personalized reads; 
+          no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, 
+          when an unknown printer took a standard dummy text.
+    </p>
+    <ul id="skillset" class="flex">
+      <li>CSS</li>
+      <li>HTML</li>
+      <li>Bootstrap</li>
+      <li>Ruby</li>
+    </ul>
+    <button type="button" class="btn-type-a">See project</button>
+  </div>`;
+  workHeaderSection.appendChild(worksCardDesktop);
+}
+worksDesktop();
+
+const cardsContainer = document.createElement('div');
+cardsContainer.className = 'grid-container';
+
+const projectArray = [
+  {
+    name: 'Professional Art<br>Printing Data',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry standard.<br>&nbsp',
+    'Featured image': {
+      mobile: {
+        src: 'images/mob_img_1.png',
+        alt: '',
+      },
+      desktop: {
+        src: 'images/desktop_img_Placeholder_1.png',
+        alt: '',
+      },
+    },
+    technologies: {
+      tech1: 'CSS',
+      tech2: 'HTML',
+      tech3: 'Javascript',
+    },
+    'Link to live version': '#',
+    'Link to source': '#',
+  },
```

Comment:
- [ ] I would kindly suggest you create a pop up modal for the first cards as well in your recent work section of your page to make it 7 cards in total. Currently, i notice the pop up functionality is not implemented for the first card. Kindly work on it. Thank you 👍 👍